src/features/academic/components/detail/GradeCard.module.scss

```
@@ -29,26 +29,37 @@
+  // 남는 가로폭 차지 + 과목명 줄바꿈 허용.
+  flex: 1;
+  min-width: 0;
 }
 
 .courseName {
   color: color.$black-100;
   @include typography.body-lg;
   @include typography.font-weight-bold;
   word-wrap: break-word;
+  overflow-wrap: break-word;
+  // flex 자식이라 줄바꿈하려면 min-width:0 필요.
```

Stylelint (17.13.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)